best-practices-metadata-and-extension.md

```diff
@@ -53,31 +60,16 @@ Two behavioral extensions are recommended that don't describe the data itself bu
     Note that the Storage Extension is still evolving and has a limited set of cloud stores defined.
     Please open a PR if your cloud store is not available yet.
 
-### Datacubes
-
-If a datacube file format (e.g. netCDF, ZARR, GRIB) is exposed, the following is recommended:
-
-- **Datacube Extension** (v2.x)
-  
-  - For a single variable: `cube:dimensions` only
-  - For multiple variables: `cube:variables` and `cube:dimensions`
-
-## Items
```

Comment:
I don't think I agree with removing "Items" and "Assets" as sections. Several of the properties below, I've never seen used in the Item and the following section would imply for me having them at the item level (data_type, offset, scale, classification:bit_fields, etc.)